File: Frontend/src/app/DTOs/Create/CreateQuestionMathematicDTO.ts
import { CreateQuestionMathematicVariableDTO } from "./createQuestionMathematicVariableDTO";

export interface CreateQuestionMathematicDTO {
    id: string;
    question: string;
    answer: string;
    digits: number;
    variables: CreateQuestionMathematicVariableDTO[];
}

File: Backend/DTOs/Create/CreateQuestionMathematicDTO.cs
namespace Learnz.DTOs;
public class CreateQuestionMathematicDTO
{
    public Guid Id { get; set; }
    public string Question { get; set; }
    public string Answer { get; set; }
    public int Digits { get; set; }
    public List<CreateQuestionMathematicVariableDTO> Variables { get; set; }
}